o2a/converter/workflow.py

```diff
@@ -31,6 +32,7 @@ def __init__(
         dag_name: str,
         relations: Set[Relation] = None,
         nodes: Dict[str, ParsedActionNode] = None,
+        task_groups: Dict[str, TaskGroup] = None,
```

Comment:
I'm wondering whether we could rename `relations` (2 lines above) to something like `task_groups_relations` (or `tg_relations`) to make it clear what kind of relations they are.

Reply:
Finally, there should be all relations that are in workflow. I would prefer not to enter additional information in the field name. I will add additional information in the class documentation.

Reply:
For now there are also `nodes` in the `Workflow` so I think having the name clearly state what kind of "relations" these are would be beneficial.

Reply:
I updated this PR.